safetab_p/README.md

```diff
@@ -40,7 +40,7 @@ All python dependencies, as specified in [requirements.txt](requirements.txt) mu
 When running locally, dependencies can be installed by running:
 
 ```bash
-sudo python3 -m pip install -r safetab_p/requirements.txt
+sudo python3 -m pip install -r <absolute path of cloned DAS_2020_DDHCA_Production_Code repository>/safetab_p/requirements.txt
```

Comment:
This approach is much harder for the user because you have to edit each path before you use it - you can't copy/paste. If all paths are relative to the repository root, I can do one `cd` and then every command will work fine without being modified.

Reply:
True, but it is slightly confusing because we'd either have to have users cd to `<absolute path of cloned DAS_2020_DDHCA_Production_Code repository>` which isn't where the README.md is located, `<absolute path of cloned DAS_2020_DDHCA_Production_Code repository>/safetab_p`. Or, we'd have to have paths with `../` to make them relative, which isn't ideal in my opinion. There are 3 total commands here, I personally like the clarity of absolute paths and don't think that the copy and paste issue is that great for 3 commands.

Reply:
We met with David and agreed to use absolute paths.